app/models/rent.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Rent < ApplicationRecord
+  belongs_to :user, foreign_key: 'user_id', inverse_of: false
```

Comment:
Do you really need `foreign_key: 'user_id', inverse_of: false` ? Think about it, please.

Reply:
removed foregin_key, rubocop demands inverse_of